<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1246</width>
    <height>647</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QVBoxLayout" name="queryLayout">
      <item>
       <widget class="QLabel" name="foodQuerylabel">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Search for Foods:</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="queryInputLayout">
        <item>
         <widget class="QLineEdit" name="queryLineEdit"/>
        </item>
        <item>
         <widget class="QComboBox" name="foodGroupComboBox">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="minimumSize">
           <size>
            <width>50</width>
            <height>0</height>
           </size>
          </property>
          <property name="editable">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QTableWidget" name="resultTableWidget">
        <property name="selectionMode">
         <enum>QAbstractItemView::SingleSelection</enum>
        </property>
        <property name="selectionBehavior">
         <enum>QAbstractItemView::SelectRows</enum>
        </property>
        <property name="verticalScrollMode">
         <enum>QAbstractItemView::ScrollPerItem</enum>
        </property>
        <property name="sortingEnabled">
         <bool>true</bool>
        </property>
        <column>
         <property name="text">
          <string>Food Name</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Description</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Food Group</string>
         </property>
        </column>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="foodRecordLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Daily Food Record:</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QPushButton" name="removeFoodPushButton">
          <property name="text">
           <string>Add Food</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="addFoodPushButton">
          <property name="text">
           <string>Remove Food </string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QTableWidget" name="recordedFoodTableWidget">
        <column>
         <property name="text">
          <string>Time</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Food Name</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Quantity</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Units</string>
         </property>
        </column>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="foodMetadataLayout">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
          <weight>75</weight>
          <italic>false</italic>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Date of Consumption:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCalendarWidget" name="foodCalendarWidget"/>
      </item>
      <item>
       <widget class="QTimeEdit" name="foodTimeEdit"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="recipeLayout">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
          <weight>75</weight>
          <italic>false</italic>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Recipe Options:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="addRecipePushButton">
        <property name="text">
         <string>Add Recipe to Daily Food Record</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="newRecipePushButton">
        <property name="text">
         <string>Create New Recipe</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="editRecipePushButton">
        <property name="text">
         <string>Edit Recipe</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="deleteRecipePushButton">
        <property name="text">
         <string>Delete Recipe</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTreeWidget" name="recipeTreeWidget">
        <property name="acceptDrops">
         <bool>false</bool>
        </property>
        <property name="sortingEnabled">
         <bool>false</bool>
        </property>
        <column>
         <property name="text">
          <string notr="true">1</string>
         </property>
        </column>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1246</width>
     <height>19</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <widget class="QMenu" name="menuAnalysis">
     <property name="title">
      <string>Analysis</string>
     </property>
     <addaction name="actionLie_to_Me"/>
     <addaction name="actionDoc_Give_it_to_me_straight"/>
    </widget>
    <addaction name="menuAnalysis"/>
    <addaction name="actionExportData"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionExportData">
   <property name="text">
    <string>Export data</string>
   </property>
  </action>
  <action name="actionExport_data">
   <property name="text">
    <string>Export data</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionLie_to_Me">
   <property name="text">
    <string>Lie to Me</string>
   </property>
  </action>
  <action name="actionDoc_Give_it_to_me_straight">
   <property name="text">
    <string>Doc. Give it to me straight.</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>actionExit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>607</x>
     <y>222</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
